shell/powershell.go

```diff
@@ -42,6 +56,18 @@ type shellStarter interface {
 
 type localShellStarter struct{}
 
+func getServerType(cmd *PowerShell) string {
+	// if any of these roles are enabled we'll detect it as 'MemberServer'
+	for _, role := range memberServerRoles {
+		// we can ignore error here, because `performExec` already logs it
+		res, _ := cmd.performExec(fmt.Sprintf(roleStatePowershellCommand, role))
```

Comment:
error is ignored, maybe we want to handle it (return it)

Reply:
done